src/mlx/warnings/warnings.py

```diff
@@ -277,6 +278,7 @@ def warnings_wrapper(args):
                         help='Possible not-used flags from above are considered as command flags')
 
     args = parser.parse_args(args)
+    Finding.fingerprints = {}  # Make sure the class variable is an empty dict
```

Comment:
Why here? Did you run into issues when executing multiple test cases?

Reply:
Resolved in e5e42e743633ba586bab1c0fad13c53371ff7f67